Cargo.toml

```diff
@@ -32,14 +36,16 @@ once_cell = "1.16.0"
 paste = "1.0.9"
 clap = { version = "4.0.29", features = ["derive"], optional = true }
 anyhow = "1.0.66"
-rodio = { version = "0.16.0", default-features = false, optional = true }
+rodio = { version = "0.16.0", default-features = false, features = ["symphonia"],  optional = true }
 cpal = { version = "0.14.2", optional = true }
 pest = "2.5.1"
 pest_derive = "2.5.1"
 rustfft = { version = "6.1.0", optional = true }
 futures-timer = { version = "3.0.2", optional = true }
 futures = { version = "0.3.25", optional = true }
 plotters = { version = "0.3.4", optional = true }
+symphonia = { version = "0.5.2", optional = true }
+parse_duration0 = { version = "3.0.0", optional = true }
```

Comment:
Why is this `parse_duration0`?

Reply:
The original `parse_duration` crate went a little overboard with features: using "big" integers, allowing negative time, etc. It was then discovered that some of these features led to [denial of service](https://github.com/zeta12ti/parse_duration/issues/21), and at the same time, the original author dropped maintainership without transferring ownership of the crate.

The author of that crate had suggested that dropping support for "big" integers and exponents would clear up the issue, and the [`parse_duration0`](https://crates.io/crates/parse_duration0) crate does just that.

Reply:
Haha, I did not know all of that history: makes sense.